From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Sean Rhodes <sean@starlabs.systems>
Date: Thu, 16 Jul 2026 23:22:00 +0100
Subject: [PATCH] util/cbmem: include endian conversion declarations

musl exposes le16toh() and le32toh() through endian.h. Include it
explicitly so the host cbmem utility builds with the Heads toolchain.

Signed-off-by: Sean Rhodes <sean@starlabs.systems>
---
 util/cbmem/cbmem.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/util/cbmem/cbmem.c b/util/cbmem/cbmem.c
index 018ea871f9..dcdc3aab6a 100644
--- a/util/cbmem/cbmem.c
+++ b/util/cbmem/cbmem.c
@@ -3,6 +3,7 @@
 #include <inttypes.h>
 #include <stdbool.h>
 #include <stdio.h>
+#include <endian.h>
 #include <stdlib.h>
 #include <string.h>
 #include <strings.h>
--
2.43.0
